source/form.html

```diff
@@ -103,19 +104,19 @@ <h2 class="visually-hidden">Форма обратной связи</h2>
           <ul class="form-impression__list">
             <li class="form-impression__item">
               <label class="form-impression__field radio">
-                <input class="radio-input" type="radio" name="overall-impression" value="Rather-positive" checked>
+                <input class="radio__input" type="radio" name="overall-impression" value="Rather-positive" checked>
```

Comment:
Нет необходимости серверу передавать пунктуацию и лексику:
```suggestion
                <input class="radio__input" type="radio" name="overall-impression" value="positive" checked>
```

2 других можно сделать `negative` и `undefined`